installer/install-clangd.cmd

```diff
@@ -0,0 +1,18 @@
+@echo off
+
+setlocal
+
+cd /d %~dp0
+
+set installer_dir=%cd%
+set server_dir=..\servers\clangd
+if exist %server_dir% rd /Q /S "%server_dir%"
+md "%server_dir%"
+cd /d "%server_dir%"
+
+echo Downloading clang and LLVM...
+curl -L -o LLVM-9.0.0-win64.exe "http://releases.llvm.org/9.0.0/LLVM-9.0.0-win64.exe"
```

Comment:
if this is MSI there might be some flags to extract it without installing.

Reply:
Seems to be NSIS installer:
```
$ file LLVM-9.0.0-win64.exe
LLVM-9.0.0-win64.exe: PE32 executable (GUI) Intel 80386, for MS Windows, Nullsoft Installer self-extracting archive
```
I'll try if `/S` works

Reply:
Awesome.

Reply:
`/S` works to run it silently, although then you don't get the option to automatically add it to PATH. There might be some option for that though.

As for extracting, I think 7zip can extract NSIS installers. @mattn Can we use 7zip instead of unzip on windows?

Reply:
does /d work? some examples at the end.

https://nsis.sourceforge.io/Docs/Chapter3.html

Reply:
`.\LLVM-9.0.0-win64.exe /S /D=some dir` does extract without user intervention, but it still triggers UAC. There might be a way to disable that, though?

Reply:
Updated batch file. Should be fully automated now.